addons/core/ACEActions.hpp

```diff
@@ -5,23 +5,23 @@ class CAManBase: Man {
             class TFAR_Radio {
                 displayName = "Radios";
                 condition = "(([] call TFAR_fnc_haveSWRadio)||([] call TFAR_fnc_haveLRRadio))";
-                exceptions[] = {"isNotInside","isNotSitting"};
+                exceptions[] = {"isNotInside", "isNotSitting", "isNotSwimming"};
```

Comment:
We want that while swimming tho.. so "isNotSwimming" ? Or am I confusing stuff?

Reply:
This way the `isNotSwimming` condition is not evaluated. This way, you can be swimming while interacting.
Ref:
https://github.com/acemod/ACE3/blob/master/addons/medical/ACE_Medical_SelfActions.hpp#L12